socotratools/socotratools/client.py

```diff
@@ -284,3 +284,17 @@ def preview_endorsement_price(self, policy_locator, endorsement_name,
                 }
 
         return self.__post("/policy/" + policy_locator + "/previewEndorsementPrice", data)
+
+    def gen_report(self, report_name, start_timestamp, end_timestamp):
```

Comment:
Our back end convention is to not use abbreviations, but I'll defer to your judgment here

Reply:
Fixed.